src/routes/+layout.svelte

```diff
@@ -128,14 +133,13 @@
 			on:editConversationTitle={(ev) => editConversationTitle(ev.detail.id, ev.detail.title)}
 		/>
 	</nav>
-	{#if currentError}
+	{#if $error === ERROR_MESSAGES.authOnly}
+		<LoginModal />
+	{:else if currentError}
```

Comment:
`LoginModal` should pop up directly, not wait until there's an error (this will also remove the need for `ERROR_MESSAGES.authOnly`)